src/RandNumGenerator.java

```diff
@@ -10,27 +10,72 @@ public static void main (String[] args) {
 
         //welcome message
         System.out.println("Welcome to Ruth's Simple Random Number Generator!");
+        System.out.println("Note, your bounds will be included in the range");
 
         //getting lower bound of range
-        System.out.println("Please input the lower bound of your range (i.e. 3)");
-        int lowerNum = mrScanner.nextInt();
-        System.out.println("The lower bound is: " + lowerNum);
-
-        //getting upper bound of range
-        System.out.println("Please input the upper bound of your range (i.e. 10)");
-        int upperNum = mrScanner.nextInt();
-        System.out.println("The upper bound is: " + lowerNum);
-
-        //getting amount of numbers
-        System.out.println("How many random numbers would you like?");
-        int amtNum = mrScanner.nextInt();
-        System.out.println("Generating " + amtNum + " number(s):");
-
-        int i = amtNum;
-        while (i > 0) {
-            int randNum = lowerNum + rand.nextInt(upperNum - lowerNum);
-            System.out.println(randNum);
-            i--;
+
+        int neg = 1;
+        int lowerNum;
+        while (neg != 0) {
+            System.out.println("Please input the lower bound of your range (i.e. 3)");
+            lowerNum = mrScanner.nextInt();
+            if (lowerNum < 0) {
+                System.out.println("You entered a negative number, please input a valid lower bound.");
+            }
+            else {
+                neg = 0;
+                System.out.println("The lower bound is: " + lowerNum);
+
+                //getting upper bound of range
+                int invalNum = 1;
+                int upperNum;
+                while (invalNum != 0) {
+                    System.out.println("Please input the upper bound of your range (i.e. 10)");
+                    upperNum = mrScanner.nextInt();
+                    if (upperNum < lowerNum) {
+                        System.out.print("Your upper bound is less than your lower bound. Please input a valid integer.");
+                    }
+                    else if (upperNum == lowerNum) {
+                        System.out.print("Your upper bound is equal to your lower bound. Please input a valid integer.");
+                    }
+                    else {
+                        invalNum = 0;
+                        System.out.println("The upper bound is: " + upperNum);
+
+
+                        //getting amount of numbers
+                        System.out.println("How many random numbers would you like?");
+                        int j = 1;
```

Comment:
`Variable naming convention`: always have meaningful names because others will have to look at your code, for this simple app, I understand what you mean, but if the app has 1K lines of code, there is no way I can determine. 
Also, consider to use `boolean` here, because you use it as a flag